### DISCUSSION TITLE
Github respond

### DISCUSSION THREAD

**Discussion by barbodmsd** [community/community#121715] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi,

I delete my repository and my contributions are gone, this this the Github respond, i don't understand what is the commit URL?

Hi Barbod,
Sorry to hear that. Could you send me a link to an example commit URL ([like this](https://github.com/octocat/Spoon-Knife/commit/d0dd1f61b33d64e29d8bc1372a94ef6a2fee76a9)) that should be appearing on your contributions but isn't? This will help me understand why it's not appearing and help you bring them back.
 
Take care,

**Comment by queenofcorgis** [comment, marked as answer]
Hi @barbodmsd , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by barbodmsd** [reply]
Hi @queenofcorgis , 
Thank you for your answer, but my problem is solved
take care,

Barbod
